src/System.IO.FileSystem/tests/Base/BaseGetSetTimes.cs

```diff
@@ -77,7 +77,7 @@ public void CanGetAllTimesAfterCreation()
         }
 
         [ConditionalFact(nameof(isNotHFS))] // OSX HFS driver format does not support millisec granularity
-        public void TimesIncludeMillisecondPart_Unix()
+        public void TimesIncludeMillisecondPart()
```

Comment:
Why are you changing the names of these methods? Shouldn't this file be reverted?

Reply:
Earlier there was 2 different tests for windows and unix. But i merged them together in https://github.com/dotnet/corefx/pull/31164 
So now this test runs on windows as well as unix too

Reply:
Ok, thanks